scripts/pylib/twister/twisterlib/config_parser.py

```diff
@@ -58,6 +58,7 @@ class TwisterConfigParser:
                        "min_ram": {"type": "int", "default": 8},
                        "modules": {"type": "list", "default": []},
                        "depends_on": {"type": "set"},
+                       "depends_on_one": {"type": "set"},
```

Comment:
having its name as `depends_on_anyof` allows to add later `depends_on_oneof` (ie. exactly one of)

Reply:
sure.